.travis.yml

```diff
@@ -1,24 +1,30 @@
 language: python
-
 matrix:
-    include:
-        - python: 2.6 # these are just to make travis's UI a bit prettier
-          env: TOXENV=py26
-        - python: 2.7
-          env: TOXENV=py27
-        - python: 3.3
-          env: TOXENV=py33
-        - python: 3.4
-          env: TOXENV=py34
-        - python: 3.5
-          env: TOXENV=py35
-        - python: nightly
-          env: TOXENV=py36
-        - python: pypy
-          env: TOXENV=pypy
-
+  include:
+  - python: 2.6
+    env: TOXENV=py26
+  - python: 2.7
+    env: TOXENV=py27
+  - python: 3.3
+    env: TOXENV=py33
+  - python: 3.4
+    env: TOXENV=py34
+  - python: 3.5
+    env: TOXENV=py35
+  - python: nightly
+    env: TOXENV=py36
+  - python: pypy
+    env: TOXENV=pypy
 install: pip install tox coveralls && tox --notest
-
 script: tox
-
 after_success: coveralls
+deploy:
+  provider: pypi
+  user: jazzband
+  distributions: sdist bdist_wheel
+  password:
+    secure: hb6wCAIduM7vMO4DNV3bQh52m9XD5nEyeMb5FN9F7MTjWwsc9jVEF8eGtE2OWM07LREVHtUgOcKW9w04JV2csQpZ6VuOsrRdjUdma13ubmNHuEEuG6vHX0iaC8D7ajNDOK2AA930QTXMR9baNF0FDxhogDvh/C9s8lyc/JFwUj4/P+1Qw1Ont/PEuilJcp3px6Gn3X2VUJXJXklCmQSYBQEfGG/jbpjd6PKxTAmAjgIXSRQFwZGU3wSt1SOy2UyXC5D4X9eFrXUz3iQBGJc9DvjBdKusBgeRP8SpSoPUluKtKjq4rHEA1NUWkXykb9npEdcEV3x0zSVPLrkr6TCQ0usWj/l0oMkirR/biQeKQkfwHzECQaXkM/lrxoBv+sGkKXi97Fm+fUiQFFYz6vFixm3DADUAu2CfCLFiYusdIUvbN8hF/Mj8SCFnZgRJHWgX9DtKXecCH8SXaECjU5ld2ogCtG3GJSLlLcvKW2twtZ+Gb8/SySIrQ3hTVyK57Yh12KQICPvO4Hii7wblj51L2crub2pefQolD5V9NMjopMuICiiOKXctnHOaC8NtTAadAVDK7t4t/wXEP6jXRZdO1oTm4pyetnyp0xiojknt8qhj66rlNIXX7jKPIChfgnRHc1ccRSHOuDmrIXgszpiZRuSvU7WkSodZwA850Hy28iY=
```

Comment:
Since it's in the public repo, I take it this needs to be combined with an additional secret in the Travis CI settings to actually be used to upload anything?